Sources/SWBProtocol/ProjectModel/ProvisioningSourceData.swift

```diff
@@ -76,7 +57,6 @@ extension ProvisioningSourceData: PendingSerializableCodable {
     public init(fromLegacy deserializer: any Deserializer) throws {
         let count = try deserializer.beginAggregate(4...5)
         self.configurationName = try deserializer.deserialize()
-        self.appIDHasFeaturesEnabled = try deserializer.deserialize()
```

Comment:
Similar to the above, we should check the count here to determine if we need to deserialize and then discard the now-unused field